sendgrid.com.domain-auth.json

```diff
@@ -0,0 +1,30 @@
+{
+  "providerId":"sendgrid.com",
+  "providerName":"Twilio SendGrid",
+  "serviceId":"domain-auth",
+  "serviceName":"Twilio SendGrid Domain Authentication",
+  "version": 1,
+  "logoUrl":"https://sendgrid.com/brand/sg-twilio/SG_Twilio_Lockup_RGBx1.png",
+  "syncPubKeyDomain": "domainconnect.sendgrid.net",
+  "syncRedirectDomain": "sendgrid.com",
+  "records":[
+    {
+      "type": "CNAME",
+      "host": "%host%",
```

Comment:
When usage of `host` is intended for subdomains then this value should be set to `@`.
Please refer to [spec](https://github.com/Domain-Connect/spec/blob/master/Domain%20Connect%20Spec%20Draft.adoc#67-hostname-in-template) for details (also examples in [6.9.4](https://github.com/Domain-Connect/spec/blob/master/Domain%20Connect%20Spec%20Draft.adoc#694-variable-examples).

Reply:
ok. all comments addressed